exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,4 @@
 const name = " Daniel  ";
+var message = "My name is " + name.trim() + " and my name is " + name.length + " characters long";
```

Comment:
Very well done on your work Hozan, impressive!
only one very personal suggestion, you could have used a variable to hold the trimmed value and then use it. That would had simplify your code. Now is a bit compacted, which again, is a personal choice. I read it better on multiple lines.

Reply:
Many thanks @Domenico-cyf for your feedback, I will consider such thing next time, as I was only trying to minimize my code.